yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -510,13 +403,7 @@ export class L2Block {
    * @returns The calldata hash.
    */
   getCalldataHash() {
```

Comment:
This thing is essentially just the hash of the body. Is there any reason that we don't add `body.hash()` and then we are simply calling that here?